content/kapacitor/v1.5/nodes/barrier_node.md

```diff
@@ -35,24 +35,31 @@ stream
   //Post the top 10 results over the last 10s updated every 5s.
   |httpPost('http://example.com/api/top10')
 ```
-> **Note:** In .delete(TRUE), TRUE must be uppercase.
+
+{{% note %}}
+In `.delete(TRUE)`, `TRUE` must be uppercase.
+{{% /note %}}
 
 ### Constructor
 
-| Chaining Method | Description |
-|:---------|:---------|
-| **barrier&nbsp;(&nbsp;)** | Create a new Barrier node that emits a BarrierMessage periodically  |
+| Chaining Method           | Description                                                        |
+|:---------                 |:---------                                                          |
+| **barrier( )** | Create a new Barrier node that emits a BarrierMessage periodically |
 
 ### Property Methods
 
-| Setters | Description |
-|:---|:---|
-| **[idle](#idle)&nbsp;(&nbsp;`value`&nbsp;`time.Duration`)** | Emit barrier based on idle time since the last received message. Must be greater than zero.  |
-| **[period](#period)&nbsp;(&nbsp;`value`&nbsp;`time.Duration`)** | Emit barrier based on periodic timer.  The timer is based on system clock rather than message time. Must be greater than zero.  |
-| **[delete](#delete)&nbsp;(&nbsp;`value`&nbsp;`Boolean`)** | Delete the group after processing each barrier. |
-| **[quiet](#quiet)&nbsp;(&nbsp;)** | Suppress all error logging events from this node.  |
-
+| Setters                                                         | Description                                                                                                                   |
+|:---                                                             |:---                                                                                                                           |
+| **[idle](#idle)&nbsp;(&nbsp;`value`&nbsp;`time.Duration`)**     | Emit barrier based on idle time since the last received message. Must be greater than zero **and the window `period`**.       |
```

Comment:
extra word, but may read better?:
Must be greater than zero **and longer than the window `period`.**

Reply:
I agree, the 'longer than' really helps.